### PR TITLE
Simplify nerv example. (#3573)

### PR DESCRIPTION
**Pull request by billymoon**
remove unneeded `examples/using-nerv/server.js` file and add small note to `README.md` to clarify that `nerv` is only used in built app, not in dev.

### CODE REVIEW

**Comment by billymoon** [comment]
Looks like the travis build did not run, just has a `null` in the logs. Can someone trigger it to run again?

File: examples/using-nerv/server.js [hunk 1]
```diff
@@ -1,29 +0,0 @@
-const port = parseInt(process.env.PORT, 10) || 3000
-const dev = process.env.NODE_ENV !== 'production'
-const moduleAlias = require('module-alias')
-
-// For the development version, we'll use React.
-// Because, it support react hot loading and so on.
-if (!dev) {
-  moduleAlias.addAlias('react', 'nervjs')
```
Comment (timneutkens) [inline examples/using-nerv/server.js]: By removing this react is used for SSR and nerv for client side rendering. Probably not what you'd want.

**Comment by timneutkens** [comment]
Besides my comment this will become obsolete when we ship v5 (🔜) https://github.com/zeit/next.js/pull/3578

File: examples/using-nerv/server.js (hunk 1 above)
Comment (billymoon) [reply examples/using-nerv/server.js]: Indeed! That could get confusing.

Thanks for the update.

**Pull request closed without merging**